test/test_checks.py

```diff
@@ -34,6 +34,18 @@ def test_noqa_doubles(self):
         self.assertEqual(list(checker.run()), [])
 
 
+class DoublesMixedTestChecks(TestCase):
```

Comment:
However, we might be better off keeping inline tests separate from multiline tests altogether (e.g. `SinglesMultilineTestChecks`)